primitives/src/crh/poseidon/parameters/tweedle.rs

```diff
@@ -0,0 +1,487 @@
+use crate::crh::{
+    PoseidonParameters,
+    FieldBasedHashParameters, PoseidonHash, batched_crh::PoseidonBatchHash,
+    PoseidonQuinticSBox,
+};
+use algebra::fields::tweedle::Fq as tweedleFq;
+use algebra::fields::tweedle::Fr as tweedleFr;
+
+use algebra::biginteger::BigInteger256;
+use algebra::field_new;
+
+#[derive(Clone)]
+pub struct TweedleFqPoseidonParameters;
+
+impl FieldBasedHashParameters for TweedleFqPoseidonParameters {
+    type Fr = tweedleFq;
+    const R:usize = 2;  // The rate of the hash function
+}
+
+impl PoseidonParameters for TweedleFqPoseidonParameters {
+
+    const T:usize = 3;  // Number of S-Boxes
+    const R_F:i32 = 4;  // Number of full rounds (for each of the two full rounds)
```

Comment:
I expect that "for each of the full rounds" means that we have 4 full rounds at the beginning, and another 4 full rounds at the end.